packages/@tailwindcss-vite/package.json

```diff
@@ -31,13 +31,15 @@
     "@tailwindcss/node": "workspace:^",
     "@tailwindcss/oxide": "workspace:^",
     "lightningcss": "catalog:",
-    "postcss-load-config": "^6.0.1",
+    "postcss": "^8.4.41",
```

Comment:
This is temporary until we start to resolve `@import` ourselves (at this point every client need this so we might as well)